src/sentry/models/grouphashmetadata.py

```diff
@@ -0,0 +1,18 @@
+from django.db import models
+from django.utils import timezone
+
+from sentry.backup.scopes import RelocationScope
+from sentry.db.models import FlexibleForeignKey, Model, region_silo_model
+
+
+@region_silo_model
+class GroupHashMetadata(Model):
+    __relocation_scope__ = RelocationScope.Excluded
+
+    # GENERAL
+    grouphash = FlexibleForeignKey("sentry.GroupHash", unique=True, related_name="metadata")
+    date_added = models.DateTimeField(default=timezone.now)
```

Comment:
If you want, you can use `DefaultFieldsModel` and you'll get `date_updated` as well

Reply:
Yeah, the problem is that this was introduced by someone that wanted to retrofit existing models, so they made the column nullable. We could make a new version that has them not null and is for use with new models only.

Reply:
Thanks for the suggestion, and TIL, but most of the data here will be added at creation time, and any updates will likely be tied to things which have their own timestamps (merges, for example), so I think we're good with just the `date_added` field.